app/src/androidTest/java/com/blockeq/stellarwallet/LaunchPage.kt

```diff
@@ -48,4 +48,16 @@ object LaunchPage : BasePage() {
                 .check(ViewAssertions.matches(ViewMatchers.isDisplayed()))
                 .perform(ViewActions.click())
     }
+
+    fun clickRecoverFromPhrase() {
```

Comment:
any methods in a class that implements POM must return the page itself.
`return this`